junebug/api.py

```diff
@@ -299,5 +303,35 @@ def get_message_status(self, request, channel_id, message_id):
         }))
 
     @app.route('/health', methods=['GET'])
+    @inlineCallbacks
     def health_status(self, request):
-        return response(request, 'health ok', {})
+        if self.config.channel_health_status:
+            channels_stuck = []
+
+            base_url = "http://%s:%s/api/queues/%s" % (
+                self.amqp_config['hostname'],
+                self.amqp_config.get('management_port', '15672'),
+                urllib.quote(self.amqp_config['vhost'], safe='')
+            )
+
+            ids = yield Channel.get_all(self.redis)
+
+            for channel_id in ids:
+                for sub in ['inbound', 'outbound', 'event']:
+                    url = "%s/%s.%s" % (base_url, channel_id, sub)
+
+                    queue = requests.get(url, auth=requests.auth.HTTPBasicAuth(
+                        self.amqp_config['username'],
+                        self.amqp_config['password'])).json()
```

Comment:
Maybe we should have a client class for using the rabbitmq management interface? There are some existing clients that we can evaluate to use, to avoid having to make our own.